README: add package managers table (#6959)
actions/runner-images: README.md

| Android NDK | 1 latest non-LTS, 2 latest LTS versions |
| Xcode     | - all OS compatible versions side-by-side <br/> - for beta, GM versions - latest beta only <br/> - old patch versions are deprecated in 3 months |

### Package managers usage

> There is a number of third-party package managers used during image generation proccess for software installation.

Reviewer: ```suggestion
We use a number of third-party package managers during image generation process for software installation.

```

> The table below reflects the list of these PMs and the software installed.

Reviewer: ```suggestion
The table below reflects the list of these package managers and the software installed.

```

Reviewer: Could you add a line here as well about how often we review this list of package managers to see if they are still secure?

Reviewer: Also any other information we can share about how we check the PMs to ensure they are secure.

### Package managers usage

We use a number of third-party package managers during image generation process for software installation.

The table below reflects the list of these package managers and the software installed.

| Operating system | Package manager | Third-party repos and packages |
| :---        |     :---:      |          ---: |
| Ubuntu      | [APT](https://wiki.debian.org/Apt) <br/> [pipx](https://pypa.github.io/pipx)        | firefox - `mozillateam/ppa` <br> git - `git-core/ppa` <br/> php - `ondrej/php` (`ubuntu-18.04` and `ubuntu-20.04` only) <br/> gcc, gfortran -  `ubuntu-toolchain-r/test` <br/><br/>ansible-core, yamllint - `pipx` <br/><br/> APT repositories: <br/><br/> [Eclipse-Timurin (Adoptium)](https://packages.adoptium.net/artifactory/deb) <br/> [Erlang](https://packages.erlang-solutions.com/ubuntu) <br/> [HHvm](https://dl.hhvm.com/ubuntu) <br/> [Mono](https://download.mono-project.com/repo/ubuntu) <br/> [PostgreSQL](https://apt.postgresql.org/pub/repos/apt) </br> [R](https://cloud.r-project.org/bin/linux/ubuntu)|

> | Windows     | [Chocolatey](https://chocolatey.org)  | No third party reposetories used     |

Reviewer 2: for the middle column, can we put package managers on its own row and in the right column separate out the tools based on the package manager we use? you'll have to use html in this case since it's a bit advanced for markdown but i think would look cleaner (since you're trying to distinguish them anyway by the suffix at the end)

| Android NDK | 1 latest non-LTS, 2 latest LTS versions |
| Xcode     | - all OS compatible versions side-by-side <br/> - for beta, GM versions - latest beta only <br/> - old patch versions are deprecated in 3 months |

### Package managers usage

We use third-party package managers to install software during the image generation process.

The table below reflects the list of these package managers and the software installed.

> The list of the repositories will be re-evauluted once a year to identify its usefulness and security state.

Reviewer 3: ```suggestion
The list of the repositories will be re-evaluated once a year to identify its usefulness and security state.
```

> The list of the repositories will be re-evaluated once a year to identify its usefulness and security state.

Reviewer: ```suggestion
The list of third party repositories is re-evaluated once a year to identify its usefulness and security state.

```

| Android NDK | 1 latest non-LTS, 2 latest LTS versions |
| Xcode     | - all OS compatible versions side-by-side <br/> - for beta, GM versions - latest beta only <br/> - old patch versions are deprecated in 3 months |

### Package managers usage

> We use third-party package managers to install software during the image generation process.
> 
> The table below reflects the list of these package managers and the software installed.
> 
> The list of third party repositories is re-evaluated once a year to identify its usefulness and security state.

Reviewer 2: ```suggestion
We use third-party package managers to install software during the image generation process. The table below lists the package managers and the software installed.
> **Note**: third-party repositories are re-evaluated every year to identify if they are still useful and secure.
```
